priv/repo/migrations/20200410202644_add_unique_index_to_cuisines.exs

```diff
@@ -0,0 +1,7 @@
+defmodule RecipeBox.Repo.Migrations.AddUniqueIndexToCuisines do
+  use Ecto.Migration
+
+  def change do
+    create unique_index(:cuisines, [:name])
```

Comment:
It's OK to have these as two separate migrations, but I personally like to collapse them down into one so that somebody reading over them can understand the table at a glance. Just a note, though, not a big enough deal to change this time around.